### ISSUE TITLE
"You changed one of the recurrences of this item, and this instance no longer exists."

### ISSUE THREAD

**Issue by Abolfazl** [aluxnimm/outlookcaldavsynchronizer#93]
I just upgraded to latest version and now I get the following error:

```
<?xml version="1.0" encoding="utf-16"?>
<SynchronizationReport xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
  <ProfileName>Main2</ProfileName>
  <ProfileId>9a746b38-b5f7-401e-b17c-5ef5f13d2ae6</ProfileId>
  <StartTime>2016-01-04T19:12:51.2009678Z</StartTime>
  <InitialEntityMatchingPerformed>false</InitialEntityMatchingPerformed>
  <ADelta>Unchanged: 0 , Added: 18 , Deleted 0 ,  Changed 0</ADelta>
  <BDelta>Unchanged: 0 , Added: 0 , Deleted 0 ,  Changed 0</BDelta>
  <LoadErrors />
  <EntitySynchronizationReports>
    <EntitySynchronizationReport>
      <MappingErrors />
      <AId>00000000416D35F8FB7AD64C868B2E9CCB24E87C0700347DE4202BF09F44AB4539B8206CD8D000089152576C00002C2973053DD98C4282C563C6C44512E0000000025C550000</AId>
      <BId />
      <ExceptionThatLeadToAbortion>System.Runtime.InteropServices.COMException (0xBFE04005): You changed one of the recurrences of this item, and this instance no longer exists. Close any open items and try again.
   at Microsoft.Office.Interop.Outlook.Exception.get_AppointmentItem()
   at CalDavSynchronizer.Implementation.Events.EventEntityMapper.MapRecurrance1To2(AppointmentItem source, IEvent target, iCalTimeZone startIcalTimeZone, iCalTimeZone endIcalTimeZone)
   at CalDavSynchronizer.Implementation.Events.EventEntityMapper.Map1To2(AppointmentItem source, IEvent target, Boolean isRecurrenceException, iCalTimeZone startIcalTimeZone, iCalTimeZone endIcalTimeZone)
   at CalDavSynchronizer.Implementation.Events.EventEntityMapper.Map1To2(AppointmentItemWrapper sourceWrapper, IICalendar existingTargetCalender, IEntityMappingLogger logger)
   at GenSync.Synchronization.States.CreateInB`6.&lt;&gt;c__DisplayClass2.&lt;PerformSyncActionNoThrow&gt;b__0(TBtypeEntity b)
   at CalDavSynchronizer.Implementation.CalDavRepository.&lt;Create&gt;d__15.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at GenSync.Synchronization.States.CreateInB`6.&lt;PerformSyncActionNoThrow&gt;d__4.MoveNext()</ExceptionThatLeadToAbortion>
    </EntitySynchronizationReport>
  </EntitySynchronizationReports>
  <Duration>00:03:21.7210266</Duration>
</SynchronizationReport>
```

The error says that " You changed one of the recurrences of this item, and this instance no longer exists."

How do I fix this?

**Comment by aluxnimm** [comment]
Seems you have an recurring Appointment in a different timezone with an exception. Outlook doesn't find it and we can't map it. Will catch that exception and ignore the lost event in the next release. you can manually remove that recurrence  exception to avoid the error.

**Comment by Abolfazl** [comment]
How can I find exactly which event it is referencing?

**Comment by aluxnimm** [comment]
Unfortunately no easy way at the moment, but we will improve the reports in a future release. You need to find it in Outlook list view for example and group by recurring events and check those manually if they have some exceptions and different timezones.

**Comment by aluxnimm** [comment]
try https://sourceforge.net/projects/outlookcaldavsynchronizer/files/1.13.1_Test/

should fix this issues.

**Comment by aluxnimm** [comment]
Fixed in 1.13.2.

**Comment by danji77** [comment]
Have this issue with the latest alpha 2.9.4.0.
